src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -13,6 +13,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_SCHEDULE = new Prefix("t/");
     public static final Prefix PREFIX_SUBJECT = new Prefix("s/");
     public static final Prefix PREFIX_RATE = new Prefix("r/");
-    public static final Prefix PREFIX_PAID = new Prefix("paid/");
+    public static final Prefix PREFIX_PAID_AMOUNT = new Prefix("paid/");
```

Comment:
Should the prefix be paid or paidAmount? In the user guide the prefix was changed to paidAmount